packages/resource-deployment/scripts/update-release-version.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# shellcheck disable=SC1090
+set -eo pipefail
+
+if [[ -z $dropFolder ]]; then
+    dropFolder="${0%/*}/../../../"
+fi
+
+exitWithUsageInfo() {
+    echo "
+Usage: $0 \
+-r <resource group> \
+-v <release version>
+"
+    exit 1
+}
+
+# Read script arguments
+while getopts ":r:v:" option; do
+    case $option in
+    r) resourceGroupName=${OPTARG} ;;
+    v) releaseVersion=${OPTARG} ;;
+    *) exitWithUsageInfo ;;
+    esac
+done
+
+getFunctionAppNames() {
+    functionAppNamesString=$(az functionapp list --resource-group "$resourceGroupName" --query '[].name' -o tsv | tr "\n" ",")
+    echo "Found function apps: $functionAppNamesString"
+    IFS=$',' read -ra functionAppNames <<<"$functionAppNamesString"
+}
+
+setReleaseVersion() {
+    functionAppName=$1
+    echo "Updating release version for function app $functionAppName"
+    az functionapp config appsettings set --name $functionAppName --resource-group $resourceGroupName --settings "RELEASE_VERSION=$releaseVersion" 1>/dev/null
```

Comment:
i would suggest to do this as part of the arm template. this way, it is consistent with how we set other env variables.

Reply:
(like in function-app-template.json)

Reply:
Are we going to do it inside the arm template?